addons/mail/static/src/messaging/component/follower_list_menu/follower_list_menu.js

```diff
@@ -0,0 +1,172 @@
+odoo.define('mail.messaging.component.FollowerListMenu', function (require) {
+'use strict';
+
+const components = {
+    Follower: require('mail.messaging.component.Follower'),
+};
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+const { useRef, useState } = owl.hooks;
+
+class FollowerListMenu extends Component {
+    /**
+     * @override
+     * @param {...any} args
+     */
+    constructor(...args) {
+        super(...args);
+        this.state = useState({
+            hasDropdown: false,
+            hasUnfollowHover: false,
+        });
+        useStore(props => {
+            const thread = this.env.entities.Thread.get(props.threadLocalId);
+            const followers = thread ? thread.followers : [];
+            const isCurrentPartnerFollowing = thread.isCurrentPartnerFollowing;
+            return [
+                followers,
+                isCurrentPartnerFollowing,
+                thread,
+            ];
+        }, {
+            compareDepth: {
+                followers: 1,
```

Comment:
I think this doesn't work with the array syntax above. `followers` here is the key/index on the returned object/array of the selector, so that should probably be `0` instead? Maybe when we use `compareDepth` we should keep the object notation in selector return just for clarity.